client/post-editor/post-editor.jsx

```diff
@@ -442,7 +442,7 @@ export const PostEditor = createReactClass( {
 							postId={ this.props.postId }
 							externalUrl={ this.getExternalUrl() }
 							editUrl={ this.props.editPath }
-							defaultViewportDevice={ this.state.isPostPublishPreview ? 'computer' : 'tablet' }
+							defaultViewportDevice={ 'computer' }
```

Comment:
Since `computer` is the default value for `defaultViewportDevice` in the `WebPreview` component, you can just delete this prop, as well as the same prop in `EditorPreview` where it calls `WebPreview`.